test/docker/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM ubuntu:18.04
 RUN apt-get update
 RUN apt-get -y install vim iptables build-essential wget git iputils-ping net-tools curl jq kafkacat apt-transport-https socat
 RUN curl -fsSL get.docker.com | sh
-RUN curl https://dl.google.com/go/go1.11.4.linux-amd64.tar.gz | tar -xzf- -C /usr/local/
+RUN curl https://dl.google.com/go/go1.14.linux-amd64.tar.gz | tar -xzf- -C /usr/local/
```

Comment:
This adjusts the e2edev container, but you also need to adjust the deb packager at this line, maybe others (in a separate PR): https://github.com/open-horizon/horizon-deb-packager/blob/master/build_support/tools/Dockerfile-render-bld.bash, and also
https://github.com/open-horizon/horizon-deb-packager/blob/master/.travis.yml and also in this project you need to update: https://github.com/open-horizon/anax/blob/master/.travis.yml

Reply:
done